responses.go

```diff
@@ -42,6 +51,14 @@ type InfoResp struct {
 	BlockCPULimit        Int64  `json:"block_cpu_limit"`
 	BlockNetLimit        Int64  `json:"block_net_limit"`
 	ServerVersionString  string `json:"server_version_string"`
+
+	EarliestAvailableBlockNum uint64         `json:"earliest_available_block_num"`
```

Comment:
Added `// Added in <version>` comment

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/76ee1c81a195b10d296659067a9f077657ea1cc4